site/class-1-bar/DesleyAalderink/readme.md

```diff
@@ -1 +1,32 @@
-# Hello World
+![Alt text][cover]
+
+ by [**@caged**][block-author]
+
+
+# Barchart!
+
+I will explain in this short description what I must do.
+
+## Background 
+
+I did this assignment for Front end 3. I will learn how to use D3. I must explain how my code will work.
+
+## Data
+
+There are two data types I use:
+
+* `Frequency` - this uses prencentages to determ how much it is.
+* `Letters`   - I use it to display the precentages with the letters. Normaly it uses all the letters in the alphabet. I customized it.
+
+## Features
+
+*  [`d3-scale`] (https://github.com/d3/d3-scale)
+*  [`d3-svg-axis`] (https://github.com/d3/d3-axis#api-reference)
+
+## License
+
+All rights belong to the user Caged. I just changed a few things from his work.
```

Comment:
Sure it’s a `he`?

Reply:
His name is Justin Palmer and he has a beard, so yes?

Reply:
Ah, okay lol